.github/workflows/triage.yml

```diff
@@ -240,3 +240,7 @@ jobs:
 
             - label: pip-audit
               pr_body_content: Created by `brew-pip-audit`
+
+            - label: autobump
```

Comment:
`- label: autobump_txt` or something feels more appropriate here?

@chenrui333 I expected from this label that it would be all the PRs that were created by the autobump workflow?